.travis.yml

```diff
@@ -7,29 +7,32 @@ notifications:
 matrix:
   fast_finish: true
   include:
-  - python: 2.7
-    env: CONDA_ENV=py27
-  - python: 3.5
-    env: CONDA_ENV=py35
   - python: 3.6
     env: CONDA_ENV=py36
+  - python: 3.7
+    env: CONDA_ENV=py37
+  - python: 3.7
+    env: CONDA_ENV=py37-upstream-master
 
 before_install:
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget http://repo.continuum.io/miniconda/Miniconda-3.16.0-Linux-x86_64.sh -O miniconda.sh;
-    else
-      wget http://repo.continuum.io/miniconda/Miniconda3-3.16.0-Linux-x86_64.sh -O miniconda.sh;
-    fi
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.16.0-Linux-x86_64.sh -O miniconda.sh;
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
   - conda info -a
+  - conda list
 
 install:
   - conda env create --file ci/environment-$CONDA_ENV.yml
   - source activate test_env_xgcm
+  # Install xarray from master not conda to be able to react to changes quicker
+  - if [[ "$CONDA_ENV" == "py37-upstream-master" ]]; then
+      pip install --no-deps git+https://github.com/pydata/xarray.git;
```

Comment:
Why not just put this in the environment file?

Reply:
Done